pkg/eval/unix/non_unix.go

```diff
@@ -0,0 +1,22 @@
+// +build  windows plan9 js
+
+// Package `unix` exposes variables and functions that deal with features
```

Comment:
Do not use backticks in godoc; they are not interpreted as markdown.

Reply:
Is there some other convention for indicating that a string is a code literal in such comments?

Reply:
In any case I've removed the backticks.